pysite/route_manager.py

```diff
@@ -19,6 +20,12 @@ def __init__(self):
         self.app = Flask(
             __name__, template_folder=TEMPLATES_PATH, static_folder=STATIC_PATH, static_url_path="/static",
         )
+
+        # Store logger in the Flask global context
+        self.log = Logs(self.app)
+        with self.app.app_context():
+            g.log = self.log  # type: logger
```

Comment:
this comment annotation is invalid. should be `# type: Logs` currently, or `# type: Logger` if you change the class name to that.